lib/token.js

```diff
@@ -12,8 +12,37 @@ function getWellKnown(sdk) {
   return http.get(sdk, sdk.options.url + '/.well-known/openid-configuration');
 }
 
-function validateClaims(sdk, claims, aud) {
-  var iss = sdk.options.url;
+function getAuthServerUrl(sdk, authServer) {
+  authServer = authServer || sdk.options.authServer;
+  if (authServer) {
+    if (/^https?:/.test(authServer)) {
+      return authServer;
+    }
+    return sdk.options.url + '/oauth2/' + authServer;
+  }
+}
+
+var isAuthorizationServer = new RegExp('^https?://.*?/oauth2/.+');
+function issuerToAuthorizationUrl(issuer) {
+  if (isAuthorizationServer.test(issuer)) {
+    return issuer;
+  }
+  return issuer + '/oauth2';
+}
+
+function getIssuer(sdk, authServer) {
```

Comment:
And maybe we don't even need authorizeBaseUrl (i.e. do we ever need to have the base without the /v1/authorize part?)